### ISSUE TITLE
Fullscreen Lion Nightly Build

### ISSUE THREAD

**Issue by fbkarsdorp** [aquamacs-emacs/aquamacs-emacs#62]
Today, (8 dec 2011) I downloaded the nightly build of Aquamacs to test out the latest development version. Concerning the fullscreen support in Lion, there still seems to be a bug. At first all went well, but after a while hitting the full screen button left me with a blank screen. Also when I tried to launch the application, it did not show up, although it was loaded in my dock. I tried reinstalling it completely (deleted the config files) but still no luck.

**Comment by fbkarsdorp** [comment]
Ok, small update. The bug seems to have something to do with closing aquamacs. I did a fresh install today (3x series) and the first run, all was fine. When I closed aquamacs, and than tried to start it again, the same problem as before happened, i.e. the application is started, but I do not see it (only after hitting command-tab) and fullscreen gives me a blank (well actually grey normal background) screen.
